packages/iov-bns/src/client.ts

```diff
@@ -58,9 +60,12 @@ function onChange<T>(): (val: T) => boolean {
   };
 }
 
-// Client talks directly to the BNS blockchain and exposes the
-// same interface we have with the BCP protocol.
-// We can embed in iov-core process or use this in a BCP-relay
+/**
+ * Talks directly to the BNS blockchain and exposes the
+ * same interface we have with the BCP protocol.
+ *
+ * We can embed in iov-core process or use this in a BCP-relay
+ */
```

Comment:
Did we want to change this name to Connection as well, or other pr I guess?

Reply:
Yes, we want. Incompatible change so I create a ticket